MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/maps/RenderViewGetterTest.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.mapboxsdk.testapp.maps
+
+import android.opengl.GLSurfaceView
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.view.TextureView
+import android.view.ViewGroup
+import com.mapbox.mapboxsdk.AppCenter
+import com.mapbox.mapboxsdk.maps.MapView
+import com.mapbox.mapboxsdk.maps.MapboxMapOptions
+import com.mapbox.mapboxsdk.testapp.activity.FeatureOverviewActivity
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+import junit.framework.Assert.*
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class RenderViewGetterTest : AppCenter() {
+
+  @Rule
+  @JvmField
+  var rule = ActivityTestRule(FeatureOverviewActivity::class.java)
+
+  private lateinit var rootView: ViewGroup
+  private lateinit var mapView: MapView
+  private val latch: CountDownLatch = CountDownLatch(1)
+
+  @Test
+  fun testGLSurfaceView() {
+    rule.runOnUiThread {
+      rootView = rule.activity.findViewById(android.R.id.content)
+      mapView = MapView(rule.activity)
+      assertNotNull(mapView.renderView)
+      assertTrue(mapView.renderView is GLSurfaceView)
+      latch.countDown()
```

Comment:
`ActivityTestRule#runOnUiThread` is blocking, no need for latches. Or even just tag the whole test as UI.